frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -280,6 +326,17 @@ export default function CourseDescription({ structure }) {
                   </Tag>
                 );
               })}
+            <Title level={3} className="text cs-final-attr">
```

Comment:
Would it be better to include the title for Capacity in the conditional rendering below? So if there is no `courseCapacity.enrolments`, then we don't show this at all (instead of showing the text `Nothing to show`)? Let me know your thoughts.

Reply:
Yeah this is better. Have put this in